dev/src/util.ts

```diff
@@ -219,27 +219,28 @@ export function wrapError(err: Error, stack: string): Error {
 }
 
 /**
- * Returns a tuple indicating if FIRESTORE_PREFER_REST is set to a valid value,
- * and if so, whether the value enables preferRest or disables preferRest.
+ * Parses the value of the environment variable FIRESTORE_PREFER_REST, and returns
+ * a value indicating if the environment variable enables or disables preferRest.
  *
- * @return A tuple where the first value indicates if the environment variable is set to a valid value,
- * and the second value indicates if the environment variable enables preferRest or disables preferRest.
+ * This method will warn to the console if the environment variable is set to an unsupported value.
+ *
+ * @return {boolean | undefined} `true` if the environment variable enables `preferRest`,
```

Comment:
nit: I'd recommend to remove the definition of the return type from the documentation since it's already specified formally in the function's return type declaration. By repeating it here you risk it getting out of sync with the actual return type, which is confusing.

Reply:
Good way to look at it.